rslib/src/scheduler/states/relearning.rs

```diff
@@ -135,8 +135,11 @@ impl RelearnState {
     fn answer_easy(self, ctx: &StateContext) -> ReviewState {
         let scheduled_days = if let Some(states) = &ctx.fsrs_next_states {
             let (mut minimum, maximum) = ctx.min_and_max_review_intervals(1);
+            // ensure 1 greater than good where possible
             let good = ctx.with_review_fuzz(states.good.interval as f32, minimum, maximum);
-            minimum = good + 1;
+            if minimum < maximum {
```

Comment:
Imo, there was no need to add this extra condition because constrained_fuzz_bounds already deals with this case.

https://github.com/ankitects/anki/blob/60b25535efaf23baa6a8a00e3509d9b43a4cdc06/rslib/src/scheduler/states/fuzz.rs#L79

Reply:
Thanks, good catch.